### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by 906836920**
add traditional chinese fixed

### CODE REVIEW

**Comment by cracky5322** [comment]
Too many variants and sentences that do not conform to traditional Chinese

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+    <string name="setting_gates">閘閥</string>
+    <string name="setting_gates_desc">停用手勢在某些情況下被識別並激活動作</string>
+    <string name="setting_feedback">反饋</string>
+    <string name="setting_feedback_desc">設定手勢運行通知</string>
+    <string name="setting_gesture_model">裝置模型</string>
+    <string name="setting_gesture_model_desc">選擇機器學習組件使用的裝置模型</string>
+    <string name="setting_feedback_vibrate">震動</string>
+    <string name="setting_feedback_vibrate_desc">使用手勢並執行動作時振動一次</string>
+    <string name="setting_feedback_wake">喚醒設備</string>
+    <string name="setting_feedback_wake_desc">當熒幕關閉時喚醒設備 (注意：這會干擾關閉熒幕的操作!)</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="setting_feedback_wake_desc">當螢幕關閉時喚醒設備 (注意：這會干擾關閉螢幕的操作!)</string>
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+    <string name="setting_actions">動作</string>
+    <string name="settings_actions_desc">設定使用手勢時要執行的動作</string>
+    <string name="setting_gates">閘閥</string>
+    <string name="setting_gates_desc">停用手勢在某些情況下被識別並激活動作</string>
+    <string name="setting_feedback">反饋</string>
+    <string name="setting_feedback_desc">設定手勢運行通知</string>
+    <string name="setting_gesture_model">裝置模型</string>
+    <string name="setting_gesture_model_desc">選擇機器學習組件使用的裝置模型</string>
+    <string name="setting_feedback_vibrate">震動</string>
+    <string name="setting_feedback_vibrate_desc">使用手勢並執行動作時振動一次</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="setting_feedback_vibrate_desc">使用手勢並執行動作時震動一次</string>
```

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
+    <string name="xposed_desc">哥倫布後門</string>
+    <string name="accessibility_service_description">TapTap 無障礙設定\n\n<b>重要提示: 應用不會收集您的個資，此服務僅用於辨識手勢活動並提供與裝置的互動支援</b></string>
+    <string name="toast_no_action_selected">尚未選擇任何動作</string>
+    <string name="launch_accessibility">啟動輔助功能</string>
+    <string name="setting_gesture">手勢</string>
+    <string name="setting_gesture_desc">調整手勢的工作方式</string>
+    <string name="setting_actions">動作</string>
+    <string name="settings_actions_desc">設定使用手勢時要執行的動作</string>
+    <string name="setting_gates">閘閥</string>
+    <string name="setting_gates_desc">停用手勢在某些情況下被識別並激活動作</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="setting_gates_desc">停用手勢在某些情況下被辨識並開始動作</string>
```

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
+    <string name="setting_feedback">反饋</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="setting_feedback">回饋</string>
```

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -26,118 +26,118 @@
...
-    <string name="action_launch_assistant">Launch Assistant</string>
-    <string name="action_launch_assistant_desc">Launch your default assistant app</string>
-    <string name="action_tasker_event">Tasker Event</string>
-    <string name="action_tasker_event_desc">Trigger the Tasker Event plugin</string>
+    <string name="gate_charging_state">充電時</string>
+    <string name="gate_charging_state_desc">當手機插入充電時，暫時停用手勢半秒</string>
+    <string name="gate_telephony_activity">通話時</string>
+    <string name="gate_telephony_activity_desc">當手機通話時，停用手勢. 僅支援標準電話 - 不支援網路電話 如.WhatsApp.</string>
+    <string name="gate_usb_state">USB 設備</string>
+    <string name="gate_usb_state_desc">當有 USB 設備插入時，停用手勢半秒鐘。例如：U盤插入</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="gate_usb_state_desc">當有 USB 設備插入時，停用手勢半秒鐘。例如：USB 隨身硬碟</string>
```

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="gate_camera_visibility">相機</string>
+    <string name="gate_camera_visibility_desc">當相機啟動時停用手勢. 注意：這將在執行動作時跳過「啟動相機」動作.</string>
+    <string name="gate_power_state">關閉熒幕</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="gate_power_state">關閉螢幕</string>
```

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="gate_power_state_desc">當熒幕關閉狀態停用手勢</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="gate_power_state_desc">當螢幕關閉狀態停用手勢</string>
```

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="gate_power_state_inverse">啟動熒幕</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="gate_power_state_inverse">啟動螢幕</string>
```

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="gate_power_state_inverse_desc">當熒幕開啟狀態停用手勢</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="gate_power_state_inverse_desc">當螢幕開啟狀態停用手勢</string>
```

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="gate_app_showing">應用程式前台</string>
+    <string name="gate_app_showing_desc">在屏幕上顯示前台應用時停用手勢</string>
+    <string name="gate_app_showing_desc_formatted">當 %1s 處於前台運行時，暫時停用手勢</string>
+    <string name="action_launch_camera">啟動相機</string>
+    <string name="action_launch_camera_desc">啟動默認相機應用. 如果相機已經在運行，這個操作將會跳過.</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="action_launch_camera_desc">啟動預設相機應用. 如果相機已啟動，這個操作將會跳過.</string>
```

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="action_home_desc">模擬按下主頁鍵</string>
+    <string name="action_back">返回</string>
+    <string name="action_back_desc">模擬按下返回鍵</string>
+    <string name="action_recents">打開最近任務</string>
+    <string name="action_recents_desc">通過模擬按下最近任務鍵來打開最近任務。在支持此功能的設備上，若最近任務已經處於打開狀態，則使用此動作會切換至第一個後台應用</string>
+    <string name="action_notifications">打開通知</string>
+    <string name="action_notifications_desc">展開通知面板</string>
+    <string name="action_quick_settings">快速設定</string>
+    <string name="action_quick_settings_desc">打開快速設定面板</string>
+    <string name="action_screenshot">擷取熒幕畫面</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="action_screenshot">擷取螢幕畫面</string>
```

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="action_screenshot_desc">使用系統擷取熒幕畫面</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="action_screenshot_desc">使用系統擷取螢幕畫面</string>
```

File: app/src/main/res/values/strings.xml [hunk 13]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="action_lock_screen">鎖定熒幕</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="action_lock_screen">鎖定螢幕</string>
```

File: app/src/main/res/values/strings.xml [hunk 14]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="action_launch_app_desc_formattable">啟動 %1s. 當該應用已處於運行狀態時，此動作將被跳過.</string>
+    <string name="fab_add_gate">增加門檻</string>
+    <string name="add_gate_category_state">狀態</string>
+    <string name="add_gate_category_action">動作</string>
+    <string name="nothing_found">什麼都沒找到</string>
+    <string name="search">搜尋</string>
+    <string name="show_system_apps">顯示系統應用</string>
+    <string name="select_an_app">選擇一個應用</string>
+    <string name="setting_gesture_sensitivity">靈敏度</string>
+    <string name="setting_gesture_sensitivity_desc">該應用當前使用的Google代碼中尚未實現敏感性。 我正在積極研究它的可能性，並且它可能在將來的版本中可用.</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="setting_gesture_sensitivity_desc">該應用目前使用的 Google 程式碼中尚未實現敏感性。 我正在積極研究它的可能性，並且它可能在將來的版本中可用.</string>
```

File: app/src/main/res/values/strings.xml [hunk 15]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="bs_help_action">當敲擊設備背面兩次時運行的任務，即為「動作」. 動作將按照列表順序從上往下執行。如果某個動作成功運行，則其他的動作不會運行。然而，若是一個動作無法運行（例如：在已經啟動了語音助手的情況下，想運行「啟動語音助手」）時，將會運行它下面的那個任務，以此類推，直到其中一個動作成功運行為止。 \n\n在該應用程式的未來版本中，您將能夠向動作添加門檻(請參閱此應用的「門檻」菜單)，以允許對某些動作何時運行進行更為靈活的控制.</string>
+    <string name="bs_help_action_title">何為動作？</string>
+    <string name="bs_help_gate">門檻可阻止手勢運行任何動作。為了運行任何動作，您必須打開所有已啟用的門檻。 \n如果當前滿足某個場景，例如：熒幕處於關閉狀態，而「熒幕關閉時」處於啟用狀態時，則不會運行任何動作。您可以添加任意數量的門檻，並使用開關來啟用或者禁用它們（沒有必要刪掉它們再重新添加），以節省時間並提高效率。 </string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="bs_help_gate">門檻可阻止手勢運行任何動作。為了運行任何動作，您必須打開所有已啟用的門檻。 \n如果當前滿足某個場景，例如：螢幕處於關閉狀態，而「螢幕關閉時」處於啟用狀態時，則不會運行任何動作。您可以添加任意數量的門檻，並使用開關來啟用或者禁用它們（沒有必要刪掉它們再重新添加），以節省時間並提高效率。 </string>
```

File: app/src/main/res/values/strings.xml [hunk 16]
```diff
@@ -26,118 +26,118 @@
...
-    <string name="accessibility_info_toast">Select "Tap, Tap" from the accessibility service list, toggle it and return to Tap, Tap to continue</string>
-    <string name="menu_alpha">This app is in Alpha. Tap for info.</string>
-    <string name="bs_alpha">This app is in the early stages of development and has been released in an "Alpha" state to allow for testing on lots of devices. If you encounter a bug, please report it on the GitHub as an issue. If you would like to request a feature, post it on the XDA thread or request it on GitHub.\n\nCheck back on XDA, either on Labs or in the thread, for updates as soon as they\'re available.\n\nThank you for your patience during development.</string>
+    <string name="accessibility_info_off">無障礙服務未啟用</string>
+    <string name="accessibility_info_off_desc">手勢無法工作. 點按以啟用</string>
+    <string name="accessibility_info_on">已啟用無障礙服務</string>
+    <string name="accessibility_info_on_desc">如果您想禁用手勢, 點按以禁用</string>
+    <string name="accessibility_info_toast">在無障礙列表選擇「Tap, Tap」並啟用它, 然後返回應用以繼續</string>
+    <string name="menu_alpha">此应用处于 Alpha 阶段，点击了解详情。</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="menu_alpha">此程式目前在 Alpha 階段，點擊了解詳細資訊。</string>
```

File: app/src/main/res/values/strings.xml [hunk 17]
```diff
@@ -26,118 +26,118 @@
...
+    <string name="bs_alpha">該應用程序仍處於早期開發階段，目前已發佈 Alpha 版本，可在眾多設備上進行測試。 若遇到錯誤, 請在Github報告錯誤。 如有新功能建議, 請將其發佈在XDA線程上或在GitHub上提交.\n\n在 XDA 上查看, 無論是在實驗室或此文章, 都將竭盡所能更新.\n\n感謝您在開發過程中的耐心等候。</string>
```
Comment (cracky5322) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <string name="bs_alpha">該程式仍處於早期開發階段，目前已發佈 Alpha 版本，可在眾多設備上進行測試。 若遇到錯誤, 請在Github報告錯誤。 如有新功能建議, 請將其發佈在XDA線程上或在GitHub上提交.\n\n在 XDA 上查看, 無論是在實驗室或此文章, 都將竭盡所能更新.\n\n感謝您在開發過程中的耐心等候。</string>
```

**Comment by KieronQuinn** [comment]
I ended up copying the strings file into a different PR as merging this would've deleted the English strings file. In future, please  make sure you just copy the strings file for localisation, _do not_ delete it, as that breaks the original.

**Comment by 906836920** [comment]
ok , i got it , i'm a github newbie,so i didn't not using it too well

Kieron Quinn <notifications@github.com> 於 2020年8月8日週六 上午1:53寫道：

**Pull request closed without merging**